### ISSUE TITLE
_firebase2.default.initializeApp is not a function

### ISSUE THREAD

**Issue by roberto68** [r-park/todo-react-redux#50]
I'm getting this error

**Comment by r-park** [comment]
Hi @roberto68
- Did you make any changes/customizations?
- When does the error occur, and where do you see it (browser console, terminal, etc)?

**Comment by roberto68** [comment]
yes I'm using the latest code - I customize it a bit but not the firebase folder /src/core/firebase where I get the error - in browser console

**Comment by roberto68** [comment]
ok now it's fine but I get
` sync.js:39Uncaught Error: Expected the routing state to be available either as `state.routing` or as the custom expression you can specify as `selectLocationState` in the `syncHistoryWithStore()` options. Ensure you have added the `routerReducer` to your store's reducers via `combineReducers` or whatever method you use to isolate your reducers. `
and also 
` VM71:1 Uncaught ReferenceError: TOGGLE is not defined `

**Comment by r-park** [comment]
Double-check your versions of these files:
- [main.js](https://github.com/r-park/todo-react-redux/blob/master/src/main.js#L14)
- [reducers.js](https://github.com/r-park/todo-react-redux/blob/master/src/core/reducers.js#L11)
- [store.js](https://github.com/r-park/todo-react-redux/blob/master/src/core/store.js)
